hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -369,7 +362,25 @@
     },
     { name: "clkmgr_aon",
       type: "clkmgr",
-      clock_srcs: {clk_i: "io_div4"},
+      clock_srcs: {
+        clk_i: "io_div4",
+        clk_main_i: {
+          group: "ast",
+          clock: "main"
+        },
+        clk_io_i: {
+          group: "ast",
+          clock: "io"
+        },
+        clk_usb_i: {
+          group: "ast",
+          clock: "usb"
+        },
+        clk_aon_i: {
+          group: "ast",
+          clock: "aon"
+        }
```

Comment:
So, if I understand correctly, main, io, usb, aon are from ast? , which also comes from external (from top_earlgrey point of view)

Reply:
yes that's right.